apps/charterafrica/.env

```diff
@@ -6,3 +6,4 @@ PAYLOAD_PUBLIC_LOCALES="en, fr, pt"
 SENTRY_ENVIRONMENT=local
 SENTRY_ORG=code-for-africa
 SENTRY_PROJECT=charterafrica
+NEXT_PUBLIC_GOOGLE_ANALYTICS_ID="G-5ZPEFLT0NJ"
```

Comment:
🤔 ... If we set the id here in `.env`, doesn't it mean even when running in dev/local, analytics events will be sent to Google?

We only want to track PROD events.

Reply:
> If we set the id here in `.env`, doesn't it mean even when running in dev/local, analytics events will be sent to Google?

I have been testing it locally over the past few days, and it looks like it does not track localhost.

Reply:
Really now? Just tested it locally now and in 2 minutes I see my events in the realtime dashboard.
<img width="1583" alt="Screenshot 2025-01-29 at 09 20 44" src="https://github.com/user-attachments/assets/c6de0189-b5c1-40ae-bc2e-1cf4a6824030" />

Reply:
I still see these GA in .env @kelvinkipruto . Why are you saying you've fixed it?

Reply:
@kilemensi I created a reusable [Google Analytics](https://github.com/CodeForAfrica/ui/blob/46a47fc263813e5303bca6fee99e1d73bcf3b5b6/packages/commons-ui-next/src/GoogleAnalytics/index.js) component that only works in prod environment

Reply:
Removed the hardcoded GA env variable